components/docs/NavToc.vue

```diff
@@ -29,14 +28,13 @@
                         </template>
                     </ul>
                 </nav>
+                <!-- <DocsNewsLetter/> -->
             </div>
         </template>
 
         <div class="d-none d-lg-block mt-4">
-            <strong class="h6 my-2 ms-3">Contribute</strong>
-            <hr class="my-2 ms-3">
-            <div>
-                <nav>
+            <strong class="h6 my-2 ms-3 title">Contribute</strong>
```

Comment:
You do use the same in content of docs, please make it a component to avoid duplication